docs/getting-started(dev)/installation.md

```diff
@@ -0,0 +1,56 @@
+---
+id: installation
+title: Installation
+---
+
+To get started on your journey with CKB, you first need to download and install the softwares you need. This includes [CKB binary](https://github.com/nervosnetwork/ckb) and [CKB-CLI](https://github.com/nervosnetwork/ckb-cli).
+
+### Download and unzip/extract the download file to an easily accessible folder.
+
+Download the binary file from the [CKB releases page on GitHub](https://github.com/nervosnetwork/ckb/releases). Please make sure the version you download must be **v0.25.2**.
+
+Open up terminal or command line.
+* On Mac:
+
+  * Either open your Applications folder, then open Utilities and double-click on Terminal, or press Command - Spacebar or Control -Spacebar to launch Spotlight and type "Terminal," then double-click the search result and the following steps are performed on Terminal.
+* On Windows:
+
+  * Note: if you are familiar with command line operation on Windows, you can skip this step and open the cmd or Power Shell terminal instead.Download Git for windows from Git-Downloads, double-click to install it and open Git Bash in start menu and the following steps are performed on Git Bash.
```

Comment:
For developers, we assume them familar with command line. So we simply skip Git, and even skip windows installation. and windows cmd usage.